### ISSUE TITLE
Can play in more than one tab at once

### ISSUE THREAD

**Issue by mwinstanley** [djmailhot/BeatTide#70]
Steps to reproduce:
Open multiple tabs. Play music in both tabs.

Possible solutions:
Use cookies and JavaScript to facilitate communication across multiple instances of the app (e.g., have each instance check the cookie's value every one second to see if the other instances have made any changes such as starting a new song--of course, this would potentially introduce concurrency issues and from there on it could be turtles all the way down. Anyway, just a thought!). --Aryan

**Comment by hsingh09** [comment]
I don't think we should prevent this behavior.  Most other website don't prevent you from doing this (you can open two tabs of grooveshark, youtube, etc.).  If the user really wants to shoot themselves in the foot and play music in two tabs at once, I say we let them as long as there aren't any consequences for the site.

**Comment by alexdmiller** [comment]
It would be cool to prevent this. But definitely low priority. 

We could also use HTML5 storage or whatever.

**Comment by hsingh09** [comment]
Will no fix for V1.  Other streaming sites (YouTube, Grooveshark, Vimeo) do not prevent streaming multiple videos or songs in different tabs.  Perhaps this is something we will implement for a later release.
